tests/test_renderers/test_docutils.py

```diff
@@ -316,6 +316,19 @@ def test_link_reference_no_key(renderer):
     )
 
 
+def test_link_reference_no_exist(renderer):
```

Comment:
This actually just replicates: https://github.com/ExecutableBookProject/MyST-Parser/blob/3d4269f4369b2fcbf22930f2a1998cb1ffdc149c/tests/test_renderers/test_docutils.py#L248

So can be removed.

To update the `test_cross_referencing` expected output to `literal` run:

```console
$ pytest -k test_cross_referencing --force-regen
```

Reply:
interesting - OK I just did that and it did indeed pass...I am a bit mystified at the test infrastructure, though it seems to work nicely haha.

Reply:
Its another on e of my tips from the top 😉 [pytest-regressions](https://pytest-regressions.readthedocs.io) is a game changer for writing certain tests

Reply:
To that point, I don't see it mentioned in the test_infrastructure page at all...is that page out of date? https://github.com/ExecutableBookProject/MyST-Parser/blob/develop/docs/develop/test_infrastructure.md

Reply:
Nope, its just a pytest plugin that adds some additional 'fixtures' to help write certain types of tests (ones that generate larger output `dict` or `str`). Its not part of the 'core' test infrastructure, and isn't necessary to write a test, but I could add a suggestion to use it.

Reply:
I think it's worth mentioning since a new developer who looks through the tests will (I assume) see syntax that is specific to that plugin, and could use help in figuring out how to understand or utilize that syntax. I'll open up an issue about it